draft-ietf-taps-transport-security.md

```diff
@@ -844,6 +861,70 @@ and the per-message nonce in the clear. Everything else is encrypted.
 
 - An unreliable transport protocol such as UDP.
 
+## OpenVPN
+
+OpenVPN {{OpenVPN}} is a commonly used protocol designed as an alternative to
+IPsec. A major goal of this protocol is to provide an easy-to-use VPN. OpenVPN
```

Comment:
"Easy-to-use" is a bit vague; is IPsec not easy in some of the same ways? Perhaps be more specific and say that `A major goal of this protocol is to provide a VPN that is simple to configure and works over a variety of transports.`

Reply:
Yep, that sounds better.